docs/integrations/saas-cloud/index.md

```diff
@@ -106,6 +106,12 @@ Learn about the Sumo Logic apps for SaaS and Cloud applications.
   <p>Monitor Delivered, Failed, Quarantined, Pending, and Spam messages.</p>
   </div>
 </div>
+<div className="box smallbox15 card">
```

Comment:
@JV0812 please change this to 16 and renumber the rest. Right now you have two # 15 cards so they're stacked on top of each other.... 

<img width="450" alt="Screenshot 2023-04-05 at 11 20 07 AM" src="https://user-images.githubusercontent.com/56411016/230169901-724086aa-92e4-4d98-a27c-8f4599548895.png">